jetty-core/jetty-siwe/src/test/resources/login.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
```

Comment:
What's the provenance of this file?

Reply:
It is used by `SignInWithEthereumEmbeddedExample`.